Mathlib/MeasureTheory/Measure/OpenPos.lean

```diff
@@ -251,3 +253,37 @@ theorem measure_closedBall_pos (x : X) {r : ℝ≥0∞} (hr : r ≠ 0) : 0 < μ
 #align emetric.measure_closed_ball_pos EMetric.measure_closedBall_pos
 
 end EMetric
+
+section MeasureZero
+/-! ## Meagre sets and measure zero
+In general, neither of meagre and measure zero implies the other.
+- for all $α ∈ (0,1)$, there is a generalised Cantor set $C ⊆ [0,1]$ of measure `α`.
+Cantor sets are nowhere dense. (Taking a countable union of fat Cantor sets whose
+measure approaches 1 yields a meagre set of measure 1.)
+- The set of Liouville numbers is a Lebesgue measure zero subset of ℝ,
+but is not meagre.
+
+However, with respect to a measure which is positive on non-empty open sets, **closed** measure
+zero sets are nowhere dense and σ-compact measure zero sets in a Hausdorff space are meagre.
+-/
+
+variable {X : Type*} [TopologicalSpace X] [MeasurableSpace X] [BorelSpace X] {s : Set X}
+  {μ : Measure X} [IsOpenPosMeasure μ]
+
+/-- A *closed* measure zero subset is nowhere dense.
+(Closedness is required: there are generalised Cantor sets of positive Lebesgue measure.) -/
```

Comment:
I'm sorry, but I still think the comment is off: Cantor sets are closed, so they are irrelevant for this lemma. Rationals, maybe?

Reply:
Oof, you're right. (It's a *different* comment now |-))
I'm sorry for taking for many rounds to get this correct. This time it should be true.